### ISSUE TITLE
File System read-only

### ISSUE THREAD

**Issue by pouryapb** [Gimenz/insta-fetcher#6]
I'm trying to use your code on a host that has only read-only access to storage, so storing cookies in a text file is making a problem.
Is there any way you could make it possible so it would use memory data only? Like an option to set when making an instance of igApi.

**Comment by Gimenz** [comment]
did you have try, put session id when making instance

```javascript
let ig = new igApi('session id here')
```

**Comment by pouryapb** [comment]
Yes, I did it exactly in this way.

**Comment by Gimenz** [comment]
oh, do you mean should i making an option to store cookie file or not?

like a 
```javascript 
new igApi('session id', false)
// choose false if didn't want  to store the cookie to file
```

**Comment by pouryapb** [comment]
Yes, or any similar solutions. Currently it can't be used in a serverless function.
It can be if I provide session ID each time the code runs.

**Comment by Gimenz** [comment]
hmmm, can you do something. and make PR 😄 
i'm not sure if i did, it will give a solution

**Comment by pouryapb** [comment]
Well I'll try but first I should read all your code 😄

**Comment by Gimenz** [comment]
ok
